.eslintrc

```diff
@@ -1,6 +0,0 @@
-{
-  "extends": "./node_modules/godaddy-style/dist/.eslintrc",
```

Comment:
Food for thought: `'use strict';` is still relevant in pure Node.js code. You could _(alternatively)_ change this to:

``` js
{
  "extends": "godaddy",
  "rules": {
    "strict": 0
  }
}
```